st2common/st2common/models/db/execution.py

```diff
@@ -70,7 +70,10 @@ class ActionExecutionDB(stormbase.StormFoundationDB):
             {'fields': ['start_timestamp']},
             {'fields': ['end_timestamp']},
             {'fields': ['status']},
-            {'fields': ['parent']}
+            {'fields': ['parent']},
+            {'fields': ['-start_timestamp']},
```

Comment:
From mongo docs

http://docs.mongodb.org/manual/core/index-compound/#sort-order

" For single-field indexes, the sort order of keys doesn’t matter because MongoDB can traverse the index in either direction. "